javascript/atoms/dom.js

```diff
@@ -312,18 +313,18 @@ bot.dom.getAttribute = function(element, attributeName) {
 /**
  * List of elements that support the "disabled" attribute, as defined by the
  * HTML 4.01 specification.
- * @private {!Array.<goog.dom.TagName>}
+ * @private {!Object<string, boolean>}
  * @const
  * @see http://www.w3.org/TR/html401/interact/forms.html#h-17.12.1
  */
-bot.dom.DISABLED_ATTRIBUTE_SUPPORTED_ = [
+bot.dom.DISABLED_ATTRIBUTE_SUPPORTED_ = goog.object.createSet([
   goog.dom.TagName.BUTTON,
```

Comment:
A toString() is added to the goog.dom.TagName class.  

Deep in createSet the argument array values are used in a property assignment:
https://github.com/google/closure-library/blob/0630ddc6697ee7921b5b125b2e882ebf5cabc028/closure/goog/object/object.js#L671
The property access converts to string:
http://www.ecma-international.org/ecma-262/6.0/#sec-topropertykey

So I think we don't need to convert toString explicitly.